php-classes/Slate/CBL/ContentAreasRequestHandler.php

```diff
@@ -84,4 +86,56 @@ public static function handleCompetenciesRequest(ContentArea $ContentArea)
             'data' => $competencies
         ]);
     }
+    
+    public static function handleRecentProgressRequest($contentArea) {
+        $student = $_GET['student'];
+        $limit = isset($_GET['limit']) ? $_GET['limit'] : 10;
+        
+        if (!$student) {
+            if (!ctype_digit($student)) {
```

Comment:
I think that I meant to use the currently logged in user as the student or something, I think the ``ctype_digit`` check would be sufficient.